starters/nextjs/firebase-ecommerce/src/app/auth/page.tsx

```diff
@@ -0,0 +1,14 @@
+import { cookies } from 'next/headers'
+import Authentication from './authentication'
+import { redirect } from 'next/navigation'
+
+export default async function AuthPage() {
+  const cookieStore = await cookies()
+  const session = cookieStore.get('__session')?.value
+
+  if (session) {
```

Comment:
we should do real auth here, you can see the latest on how to accomplish this in nextjs in the updates I made to our codelab https://github.com/firebase/friendlyeats-web/blob/bffaeae7647f7ea61b4a369a9567881cb8183e8c/nextjs-end/src/components/Header.jsx#L12